src/Controller/EnMarche/MapController.php

```diff
@@ -27,10 +27,6 @@ public function committeesAction()
         ]);
     }
 
-    /**
-     * @Route("/evenements/la-carte", name="map_events")
-     * @Method("GET")
-     */
     public function eventsAction()
```

Comment:
Please add a comment to explain that the corresponding route in define in the config unlike the other actions of this class.